solidity/random-beacon/test/system/e2e.test.ts

```diff
@@ -167,6 +174,7 @@ describe("System -- e2e", () => {
             txSubmitRelayEntry.blockNumber,
             noMisbehaved
           )
+          activeGroupMembers.push(dkgResult.members)
```

Comment:
I don't understand fully what this test does. Could you please clarify, why are we pushing new entries to the `activeGroupMembers` array? With each loop, we're adding more entries, shouldn't we just set the array completely `activeGroupMembers = dkgResult.members`?

Reply:
We have 15 requests and for every 5th request, there's a new dkg that happens. In total, we'll have 3 groups that can be chosen to participate in dkg. It mimics the group selection logic which is implemented in `selectGroup()`. In this example, say 12th request will choose a group number 1 and we need to keep track of all the groups (and members inside) so we can pass those members in `submitRelayEntry()`. That's why we have a `groupMembers` array here.